### PR TITLE
Manage unavailable history for mounts/stores

### PR DESCRIPTION
**Pull request by yme44**
Currently, gopass does not support history for mounts/stores (at least
on Linux). As a consequence, there is an exception which avoid the
display of secrets for mounts/stores.

Now, all history exceptions are catched and the history is empty.

Note: I create this commit from 0.6.0 because of compilation problems with 0.7.0 (on Linux).

### CODE REVIEW

File: src/renderer/secrets/Gopass.ts [hunk 1]
```diff
@@ -22,19 +22,23 @@ export default class Gopass {
...
+                    const lineSplit = historyLine.split(' - ')
+
+                    return {
+                        hash: lineSplit[0],
+                        author: lineSplit[1],
+                        timestamp: lineSplit[2],
+                        message: lineSplit[3]
+                    }
+                })
+        } catch (e) {
```
Comment (ruettenm) [inline src/renderer/secrets/Gopass.ts:39]: Thanks for the PR 👍

If we don’t take care about the exception I would implement it like:

```
} catch {
    return []
}
```

**Comment by ruettenm** [comment]
There are merge conflicts, could you please merge the latest master into your branch?

**Comment by yme44** [comment]
Conflict solved and your comment has been integrated.

**Comment by jverhoelen** [comment]
Thanks @yme44 👍 Merged, it will be integrated into the next release which I try to do today

**Pull request merged**